source/code-of-conduct/index.html.haml

```diff
@@ -1,44 +1,39 @@
 ---
-title: Brug - Code of Conduct - Ruby Belgium
+title: Code of Conduct - Ruby Belgium
 ---
 
-%header.header
+%header
```

Comment:
I don't believe it's a bad practice, because context is already there, thanks to the _cascading_ nature of CSS, and the hierarchical nature of HTML tag:

- To have a default style for all page headers: `body > header`.
- To have a non-default style for one page header: `body.code-of-conduct > header`.
- To have a style for all sub-header sections: `body section > header` (we don't have that case yet, I believe).

Simple, clean, readable code. No drawbacks. Uses HTML standards as designed.